add a test for accounts scenario

qnkhuat opened the pull request:
Ref: https://github.com/metabase/metabase/issues/50261

qnkhuat committed:
add a test for accounts scenario

qnkhuat committed:
cljfmt

qnkhuat committed:
people won't like this hehe

crisptrutski commented on src/metabase/search/config.clj:
```diff
@@ -77,7 +77,7 @@
 (def models-search-order
   "The order of this list influences the order of the results: items earlier in the
   list will be ranked higher."
-  ["dashboard" "metric" "segment" "indexed-entity" "card" "dataset" "collection" "table" "action" "database"])
```
This is also going to affect legacy search, but I think it's fine - quite a minor difference in the score anway.

crisptrutski commented on test/metabase/search/scenarios_test.clj:
```diff
@@ -0,0 +1,54 @@
+  [search-string]
+  (map (juxt :model :id :name) (search.tu/search-results search-string {:search-engine "fulltext"})))
+
+(deftest e2e-accounts-test
```
It would be nice if the test more closely represented the variation in the actual stats data, so that its less tied to our current rankers and their parameters.

For example:

- Using the actual view counts rather than numbers around our magic number 40.
- Having multiple text matches in some of the names, having descriptions, varying length of unique words in there.
- Having some pinned items (even though our weight was changed to 0)

crisptrutski reviewed (approved):
Not going to block on this, worth just merging if you're working on notifications for a bit.

qnkhuat committed:
Merge branch 'master' into search-accounts-test

crisptrutski committed:
Send separate events for CSV appending and replacing (#50525)

crisptrutski committed:
Enable New Search by default (#50520)

lbrdnk committed:
[Databricks] Add catalog validation into `can-connect?` implementation (#50468)

* Add catalog validation in can-connect? implementation

* Remove redundant let

* Adjust description

* Update modules/drivers/databricks/src/metabase/driver/databricks.clj

Co-authored-by: metamben <103100869+metamben@users.noreply.github.com>

---------

Co-authored-by: metamben <103100869+metamben@users.noreply.github.com>

crisptrutski committed:
Basic test for search reindexing (#50560)

npretto committed:
fix(sdk): make modals use the correct portal (#50565)

---------

Co-authored-by: Mahatthana (Kelvin) Nomsawadi <me@bboykelvin.dev>

deniskaber committed:
feat(sdk): Add cross-version e2e tests using a published SDK package (#50423)

metamben committed:
Implement ISO day of week extraction (#50475)

* Implement ISO day of week extraction

Fixes #50424

jeff-bruemmer committed:
docs - notes on transparent embeds (#50377)

camsaul committed:
Fix building with Java 11 and stop using Depstar (#50062)

crisptrutski committed:
Fix CSV upload when materialized views are present (#50557)

Fixes https://github.com/metabase/metabase/issues/50501

adam-james-v committed:
Backend Dependency Bump (#46309)

Co-authored-by: Alexander Solovyov <alexander@solovyov.net>
Co-authored-by: lbrdnk <lbrdnk@users.noreply.github.com>
Co-authored-by: Ngoc Khuat <qn.khuat@gmail.com>

heypoom committed:
fix(sdk): entity picker theming fixes (#50449)

* use css variables for skeleton background color

* apply theme to folder color

* replace theme.white with themeColor

* use name of folder chevron color

* add background-inverse color mapping

* Revert "add background-inverse color mapping"

This reverts commit d0b546c58313fd4f875ef9fdec3cad92236edfc1.

* avoid overriding folder chevron color in emotion

* use Mantine style overrides for NavLink

heypoom committed:
feat(sdk): make tooltips themeable (#50457)

* add background-inverse color mapping

* add tooltip themes

* tooltip theming

* apply to echarts tooltip

* update tooltip default color

* update tooltip defaults

* add sdk color mapping

* add additional theme colors

* add echarts tooltip story

* add loki tests

* remove background-inverse theme color option

* add to docs

* use ChartTooltipRoot for story and update loki snapshot

* update secondaryTextColor docstring

* fix unit tests

deniskaber committed:
Fix issue with cypress config for e2e cross version tests (#50592)

adam-james-v committed:
Table Column Ordering Should Not Prevent Pivot Exports (#50599)

* Table Column Ordering Should Not Prevent Pivot Exports

Prior, when viz-settings existed for table column ordering, or if some columns were hidden in a regular table viz, the
pivot export could fail.

Now, if a pivot table is being exported, the table column sorting is properly ignored and the pivot export should work.

* address feedback

sansolovyov committed:
update bigquery and netty (#50588)

alxnddr committed:
Sankey polish (#50550)

* do not show column selection error on empty datasets

* persist default sankey column settings when saving

* add more specs to chart-definition logic

* more specs

sansolovyov committed:
skip META-INF/LICENSE since it conflicts with other licenses on MacOS (#50638)

heypoom committed:
ci(sdk): remove unreliable loading indicator e2e for create question (#50555)

alxnddr committed:
simplify use-license (#50613)

* simplify use-license

* rip moment

appleby committed:
Enable underlying-records drill in table viz for multi-stage queries (#50556)

* Enable underlying-records drill in table viz for multi-stage queries

* Fix e2e test

* Add tests for underyling-records drill for multi-stage queries

* Review feedback: filterv not-empty etc

* Review feedback: add a breakout-only query to drill-thru canned-queries

deniskaber committed:
Fix slack post recurrence field label (#50666)

qnkhuat committed:
test fixes

qnkhuat committed:
Merge branch 'master' into search-accounts-test

qnkhuat committed:
Merge branch 'master' into search-accounts-test

crisptrutski commented:
Would be nice to get this merged today

qnkhuat commented:
CI is having a hard time today :(

qnkhuat committed:
Merge branch 'master' into search-accounts-test

qnkhuat committed:
Merge branch 'master' into search-accounts-test

qnkhuat committed:
Merge branch 'master' into search-accounts-test

qnkhuat committed:
Merge branch 'master' into search-accounts-test

crisptrutski committed:
kick ci
